### PR TITLE
Add a policy for blacklisting GATT UUIDs.

### LINKED ISSUE #10: Sensible Blacklisting practices?
I'm currently developing [a product](http://www.puck-js.com/) which I'm hoping will be used heavily with Web Bluetooth.

I had planned on updating firmware over BLE (obviously allowing any device to update firmware was insecure, so it would have needed physical access to the device to have put it into bootloader mode). Unfortunately I've just found out that this is now blocked _because it might be insecure on some devices_ and I'll have to wait for a new bootloader version from Nordic.

I already own maybe 10 devices that implement DFU, and on these it was done in a sensible way that wouldn't have been a security problem. These can no longer be updated by Web Bluetooth. There are probably over 100,000 devices like this out there that can no longer be updated.

It's even more worrying as I was on the cc list of the emails about getting Web Bluetooth and OTA updates working, and yet even then I had no idea this was now blocked until a few days ago.

I also use the Nordic UART UUIDs for a UART service, so that I can be compatible with existing apps that use the service.

My worry is that at some point in the future, a manufacturer is going to do something dodgy like make a pacemaker with a Nordic UART UUID (which let's face it, is quite likely) and then you will feel compelled to block that UUID in Web Bluetooth - breaking it for everyone who is using that service (it's not just me - Adafruit use it on their BLE products and apps, as do many others).

Or what if someone cloned a legitimate product's UUID, and allowed someone to do something dangerous with it. Are you then going to block even the legitimate product?

How can we ensure this doesn't happen, or that people are notified if it is about to happen?

Can we have a flag, or a black-whitelist that allows certain URLs to keep using the blacklisted UUIDs if their product depends on it?

If any manufacturer can have their product remotely disabled at any time without notice, it's going to be hard to convince anyone to invest the time to use Web Bluetooth for anything serious.

### LINKED ISSUE #258: What is a secure Device Firmware Update Service?
Raised by https://github.com/WebBluetoothCG/registries/issues/10, we need to define what makes a secure Device Firmware Update (DFU) GATT service that will not blacklisted by Web Bluetooth.

This thread is an ongoing discussion whose goal is to list Secure DFU requirements.

### PR DESCRIPTION
**Pull request by jyasskin**
Fixes #10, fixes WebBluetoothCG/web-bluetooth#258.

@beaufortfrancois @gfwilliams

### CODE REVIEW

File: gatt_blacklist_policy.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Policy for adding UUIDs to the GATT Blacklist
+
+## UUIDs are added through Github issues in this repository
+
+There is currently no minimum or maximum time between
+when the issue is filed and when the new UUID is added to the blacklist.
```
Comment (beaufortfrancois) [inline gatt_blacklist_policy.md:6]: Should we mention the finch blacklist that allows Chrome to deploy very quickly in case of an emergency?

**Review by beaufortfrancois — CHANGES_REQUESTED** [review]
LGTM but nit

File: gatt_blacklist_policy.md (hunk 1 above)
Comment (jyasskin) [reply gatt_blacklist_policy.md:6]: How's this? I don't really want to mention Chrome specifically, but all implementations should have some way to push these changes quickly.

**Review by beaufortfrancois — APPROVED** [review]
LGTM

**Pull request merged**